CourseSearchBuilderBlobTrigger/__init__.py

```diff
@@ -0,0 +1,65 @@
+import os
+import logging
+import requests

```

Comment:
requests doesn't seem to be used?

Reply:
Forgot to remove once I had refactored code